docs/start/tutorial.md

```diff
@@ -1814,7 +1813,9 @@ The fetcher knows the form data being submitted to the action, so it's available
 function Favorite({ contact }) {
   const fetcher = useFetcher();
 
-  let favorite = contact.favorite;
+  const favorite = fetcher.formData
+    ? fetcher.formData.get("favorite") === "true"
+    : contact.favorite;
   if (fetcher.formData) {
     favorite = fetcher.formData.get("favorite") === "true";
   }
```

Comment:
```suggestion
```

This code needs to be removed if you make the above a `const`

Reply:
Good catch ! Thank you. I fixed this (and another occurence of `let`).